### PR TITLE
Add sql tests for covar/corr and regr functions

### PR DESCRIPTION
**Pull request by anusudarsan**

### CODE REVIEW

**Comment by anusudarsan** [comment]
@akshatnair @brian-rickman can you please review?

File: presto-product-tests/src/test/resources/sql-tests/testcases/aggregate/corr.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- database: presto; groups: mytest; tables: orders
+select round(corr(o_totalprice, o_orderkey),6) from orders
```
Comment (akshatnair) [inline presto-product-tests/src/test/resources/sql-tests/testcases/aggregate/corr.sql:2]: "round" is needed as there is no decimal support yet?

Comment (anusudarsan) [inline presto-product-tests/src/test/resources/sql-tests/testcases/aggregate/corr.sql:2]: nope, there was mild precision diff (after 8 or 9 decimal points) in results comparing psql baseline n postgres. So added a round.

**Comment by akshatnair** [comment]
lgtm, but once we have decimal we should remove round

**Pull request closed without merging**